src/oci-config.h

```diff
@@ -1,18 +1,18 @@
 /*
  * This file is part of cc-oci-runtime.
- * 
+ *
  * Copyright (C) 2016 Intel Corporation
- * 
+ *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
  * as published by the Free Software Foundation; either version 2
  * of the License, or (at your option) any later version.
- * 
+ *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
- * 
+ *
```

Comment:
Please remove this serie of whitespace cleanup, it adds noise to the patch and makes it more difficult to read/review.